config/Dockerfiles/tests.d/libvirt/05_libvirt-builtins

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash -xe
+
+# Verify basic provisioning with global hooks
+# distros.exclude: centos7
+# providers.include: libvirt
+# providers.exclude: none
+
+set -o pipefail
+
+DISTRO=${1}
+PROVIDER=${2}
+
+TEST_NAME="${DISTRO}/${PROVIDER}/05_libvirt-builtins"
+DESCRIPTION="Test ${PROVIDER} provider provisioning with builtin hooks"
+
+echo "Test Name: ${TEST_NAME}"
+echo "${DESCRIPTION}"
+
+TARGET="libvirt-builtins"
+
+pushd docs/source/examples/workspaces/libvirt-global-hooks
+
+echo "Testing check the builtins port_up"
+
+find / -name "*port_up.yaml"
```

Comment:
what is the purpose of the find?  Is this left over debug?

Reply:
Its a left over from debug. I will remove it :+1: good catch.